app/scripts/lib/setupSentry.js

```diff
@@ -146,31 +154,125 @@ export default function setupSentry({ release, getState }) {
       } else {
         return null;
       }
-      return breadcrumb;
+      const newBreadcrumb = removeUrlsFromBreadCrumb(breadcrumb);
+      return newBreadcrumb;
     },
   });
 
-  function rewriteReport(report) {
-    try {
-      // simplify certain complex error messages (e.g. Ethjs)
-      simplifyErrorMessages(report);
-      // modify report urls
-      rewriteReportUrls(report);
-      // append app state
-      if (getState) {
-        const appState = getState();
-        if (!report.extra) {
-          report.extra = {};
-        }
-        report.extra.appState = appState;
+  return Sentry;
+}
+
+/**
+ * Receives a string and returns that stringif it is a
+ * regex match for a url with a `chrome-extension` or `moz-extension`
+ * protocol, and an empty string otherwise.
+ *
+ * @param {string} url
+ * @returns String
```

Comment:
An improvement to the docs here:
```suggestion
 * @param {string} url - The URL to check.
 * @returns {string} An empty string if the URL was internal, or the unmodified URL otherwise.
```